src/analyzer/protocol/rdp/rdp-analyzer.pac

```diff
@@ -154,6 +155,29 @@ refine flow RDP_Flow += {
 		return true;
 		%}
 
+
+        function proc_rdp_client_cluster_data(ccluster: Client_Cluster_Data): bool
+                %{
+                if ( ! rdp_client_cluster_data )
+                        return false;
+
+                RecordVal* ccld = new RecordVal(BifType::Record::RDP::ClientClusterData);
+		ccld->Assign(0, val_mgr->GetCount(${ccluster.flags}));
+		ccld->Assign(1, val_mgr->GetCount(${ccluster.redir_session_id}));
+
+		ccld->Assign(2, val_mgr->GetBool(${ccluster.REDIRECTION_SUPPORTED}));
+		ccld->Assign(3, val_mgr->GetCount(${ccluster.SERVER_SESSION_REDIRECTION_VERSION_MASK}));
+		ccld->Assign(4, val_mgr->GetCount(${ccluster.REDIRECTED_SESSIONID_FIELD_VALID}));
+		ccld->Assign(5, val_mgr->GetBool(${ccluster.REDIRECTED_SMARTCARD}));
```

Comment:
These 4 fields aren't defined in the new `RDP::ClientClusterData` record you defined ?  Either add them there, or remove these assignments if not needed.